lib/modules/manager/pixi/artifacts.ts

```diff
@@ -0,0 +1,106 @@
+import is from '@sindresorhus/is';
+import { TEMPORARY_ERROR } from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type { ExecOptions } from '../../../util/exec/types';
+import {
+  deleteLocalFile,
+  ensureCacheDir,
+  getSiblingFileName,
+  readLocalFile,
+  writeLocalFile,
+} from '../../../util/fs';
+import { Result } from '../../../util/result';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+import { pickConfig } from './lockfile';
+import { LockfileYaml } from './schema';
+
+export const commandLock = 'pixi lock --no-progress --color=never --quiet';
+
+export async function updateArtifacts({
+  packageFileName,
+  updatedDeps,
+  newPackageFileContent,
+  config,
+}: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
+  logger.debug(`pixi.updateArtifacts(${packageFileName})`);
+  const { isLockFileMaintenance } = config;
+
+  if (!is.nonEmptyArray(updatedDeps) && !isLockFileMaintenance) {
+    logger.debug('No updated pixi deps - returning null');
+    return null;
+  }
+
+  const lockFileName = getSiblingFileName(packageFileName, 'pixi.lock');
+  const existingLockFileContent = await readLocalFile(lockFileName, 'utf8');
+  if (!existingLockFileContent) {
+    logger.debug(`No lock file found`);
+    return null;
+  }
+  logger.trace(`Updating ${lockFileName}`);
+
+  const cmd: string[] = [commandLock];
```

Comment:
```suggestion
  const cmd = [commandLock];
```

typescript knows it's a string array

Reply:
yes it's a legacy type hint, it used to be `cmd.push(commandLock)`